src/twisted/web/http.py

```diff
@@ -1151,6 +1154,11 @@ def write(self, data):
 
             for name, values in self.responseHeaders.getAllRawHeaders():
                 for value in values:
+                    if (self.method == b"CONNECT" and self.code // 100 == 2 and
+                        name == b"Content-Length"):
+                        self._log.info("Warning: Removing Content-Length"
```

Comment:
1. Can we include some indication of *which* request as a parameter to this message?  Include the URL, or a reference to which render method was invoked, or both?  This log message should be actionable; which method set this header, so I can go fix it?
2. If this is a warning, don't say `Warning:`, just use `log.warn` :)  (This will also help make it more actionable, since if we're filtering out `info`-level messages that don't need to be acted upon, we'll still see it.)
3. can you test the emission of this log message?
4. can you include a reference to the relevant RFC here?